.github/workflows/releaser.yml

```diff
@@ -160,6 +160,13 @@ jobs:
           GH_ARGS: --repo=${{ github.server_url}}/${{ github.repository }}
           GH_TOKEN: ${{ github.token }}
 
+      - name: Checkout the repository to update the nightly tag
+        if: env.NIGHTLY_RELEASE == 'true'
+        uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # pin v4.1.5
```

Comment:
I don't understand how the next command (line 173&174) worked before this PR since the repo wasn't checked out and the command does a `git tag` & `git push` :thinking:

Reply:
Before this PR those lines don't work as we were not in a git repository.